platform/core-api/core-api.iml

```diff
@@ -22,5 +22,6 @@
     </orderEntry>
     <orderEntry type="library" name="asm" level="project" />
     <orderEntry type="library" name="CGLIB" level="project" />
+    <orderEntry type="library" name="Guava" level="project" />
```

Comment:
Yes, we have Guava in Intellij but this module is quite low level one and AFAIK used in several projects so we need to check all of them to have Guava.